src/SelectInput.tsx

```diff
@@ -82,7 +83,7 @@ function SelectInput<V>({
 	const previousItems = useRef<Array<Item<V>>>(items);
 
 	useEffect(() => {
-		if (!isEqual(previousItems.current, items)) {
+		if (!isEqual(_map(previousItems.current, 'value'), _map(items, 'value'))) {
```

Comment:
Could you use native `.map()` function over the one in lodash?

Reply:
Changed. But there seems to be a style issue.

Reply:
Fixed with `prettier`